subprojects/flatbuffers.wrap

```diff
@@ -1,7 +1,10 @@
 [wrap-file]
-directory = flatbuffers-1.11.0
-source_url = https://github.com/google/flatbuffers/archive/v1.11.0.tar.gz
-source_filename = flatbuffers-1.11.0.tar.gz
-source_hash = 3f4a286642094f45b1b77228656fbd7ea123964f19502f9ecfd29933fd23a50b
+directory = flatbuffers-2.0.0
+source_url = https://github.com/google/flatbuffers/archive/refs/tags/v2.0.0.tar.gz
+source_filename = flatbuffers-2.0.0.tar.gz
+source_hash = 9ddb9031798f4f8754d00fca2f1a68ecf9d0f83dfac7239af1311e4fd9a565c4
 patch_directory = flatbuffers
 
+[provide]
+flatbuffers = flatbuffers_dep
+program_names = flatc, flathash
```

Comment:
Program_names requires you to perform `meson.override_find_program()` in the meson.build file.
(Available since meson 0.46.0.)

Reply:
I thought that was in your local projects meson build file to override the program on the system/path with the one in a provided plugin?

Reply:
Well, you could, but then it would have to be done via subproject.get_variable() *instead of* a provide section.